src/public/js/github/orgs.js

```diff
@@ -0,0 +1,38 @@
+import request from '../utils/request.js';
+import login from '../login.js';
+import repos from './repos.js';
+
+export default (orgs) => {
+  document.querySelector('#app').innerHTML = `
+    <div>
+      <h4>Organisations</h4>
+      <div class="columns organisations">
+        ${
+          orgs.map((org) => {
+            return `
+              <div id="${org.id}" name="org" class="column organisation">
+                <img src="${org.image}"><span>${org.name}</span>
+              </div>
+            `;
+          }).join('')
+        }
+      </div>
+    </div>`;
+
+  const orgNodes = document.querySelectorAll('div[name=org]');
+  [].forEach.call(orgNodes, (org, i) => {
+    org.addEventListener('click', () => {
+      const orgId = orgNodes[i].id;
+      request.get(`/orgs/${orgId}`)
```

Comment:
In that case could you update the wiki to reflect that https://github.com/gitodoro/gitodoro/wiki/repos